client/blocks/site/style.scss

```diff
@@ -71,8 +71,15 @@
 	position: relative;
 	width: 100%;
 
-	&:focus {
+	.accessible-focus &:focus {
 		outline: none;
+		box-shadow: inset 0 0 0 2px var( --color-primary-light );
```

Comment:
The focus ring is overlapped by the long-content-fade on the right side, with an unfavorable result:

<img width="282" alt="Screenshot 2019-04-26 at 15 00 41" src="https://user-images.githubusercontent.com/664258/56809456-2cc1d780-6834-11e9-8401-105fe599a6ce.png">

Reply:
Nice catch! Updated in: https://github.com/Automattic/wp-calypso/commit/dd1f099f8760ed332fc504be86b7c927f3ea41b9

Here's a quick preview of what it looks like now:

![](https://d1jfzjx68gj8xs.cloudfront.net/items/2A2z093W0e3p1j0d1n3v/Screen%20Recording%202019-04-26%20at%2010.10%20AM.gif)